config/checkstyle_checks.xml

```diff
@@ -740,6 +740,12 @@
       <property name="format" value="[^\p{ASCII}]"/>
       <property name="ignoreComments" value="true"/>
     </module>
+    <module name="RegexpSinglelineJava">
+      <property name="id" value="noUsageOfGetFileContents()Method"/>
+      <property name="format" value="getFileContents()|getFileContents\s+(\s+)"/>
```

Comment:
Where do we have spaces between the method name and the argument list?

Reply:
In addition, if we are going this route why don't we use https://github.com/sevntu-checkstyle/sevntu.checkstyle/blob/master/sevntu-checks/src/main/java/com/github/sevntu/checkstyle/checks/coding/ForbidCertainMethodCheck.java ?

Reply:
> Where do we have spaces between the method name and the argument list?

Actually nowhere :) I kinda forgot we have checks for that. Will remove this pattern.

Reply:
> In addition, if we are going this route why don't we use https://github.com/sevntu-checkstyle/sevntu.checkstyle/blob/master/sevntu-checks/src/main/java/com/github/sevntu/checkstyle/checks/coding/ForbidCertainMethodCheck.java ?

We can use ForbidCertainMethodCheck for this. But I actually don't see any major superiority of ForbidCertainMethodCheck over RegexpSinglelineJava in this case. If you see any, please explain.

Reply:
I was just bringing it as an option. Its best to examine both to determine what you are trying to accomplish and what issues you will go with either option and coming to a good conclusion.

RegExp is a text only parser. It can create false positives on non-code like Javadocs and can create false negatives when the code doesn't match the exact lining and spacing. Put a line between your code and your parsing misses it.

With the Java parser, you know the specific type of code you are hitting and don't have to worry about the above. However it sometimes longer to build out a specific check to use the Java parser while regexp is usually faster.

Reply:
Regexp might be easy approach.
We can use xpath Check for this too to make it more strict and work only on Checks that extend certain classes.
But I am not sure if it worse it.